app/src/data/ApiWrapper.ts

```diff
@@ -198,6 +200,123 @@ export class ApiWrapper {
         return await this.hostLanguageForm.putResponse(item)
     }
 
+    // TODO: Hassen
```

Comment:
Better practice here is to use `Promise<void>` so we explicitly call out that no return value is expected